### ISSUE TITLE
Can't POST when model has relationship that isn't set, even though it's not required in loopback model

### ISSUE THREAD

**Issue by tsteuwer** [digitalsadhu/loopback-component-jsonapi#42]
So, in loopback, I made relationships where a `Person` can have an `Address" (required: false). However, if I want to create the `Person` without the `Address`, I can't.

**POST Request**
```
data: {
   attributes: {
      first-name: "Joe",
      last-name: "Shmoe",
      age: 31,
      created-at: "2015-10-31T14:24:02.918Z"
   },
   relationships: {
      address: {
         data: null
      }
   }
   type: "people"
}
```

**Response**
```
{ 
   errors: [
      {
         status: 500,
         detail: "Cannot read property 'type' of null"
      }
     ]
}
```

What I believe is happening is it's trying to add the relationship, even though the data is null (aka, an empty relationship as the loopback model does not require it).

**Comment by listepo** [comment]
@tsteuwer see ```master```

**Comment by tsteuwer** [comment]
Derp. For some reason I thought I the last release fixed this.... Lol. Closing.

**Comment by digitalsadhu** [comment]
The npm package is behind. I'll publish a new version.

**Comment by tsteuwer** [comment]
Is there a way to automate the npm package to stay up to date if a new release is created?

**Comment by digitalsadhu** [comment]
ok just published v0.6.0 which is up to date with master.

**Comment by digitalsadhu** [comment]
@tsteuwer if you specify the version with the caret operator and run npm install as part of some automated setup it should stay up to date.

eg. "loopback-component-jsonapi": "^0.6.0"
